modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc

```diff
@@ -1,25 +1,65 @@
 = Usage Metrics
 :description: Learn about the metrics Redpanda uses to measure consumption in Redpanda Cloud.
+:page-cloud: true
 
-Redpanda Cloud billing is mapped to the consumption of resources. When you create a cluster, you select a xref:deploy:deployment-option/cloud/cloud-overview.adoc#cluster-tiers[cluster tier] that specifies the expected throughput for your cluster, including the maximum ingress, egress, partitions, and connections. Billing is based on the resources consumed, multiplied by the cost of that resource.
+Redpanda Cloud billing is mapped to the consumption of resources. Billing is based on the resources consumed, multiplied by the cost of that resource.
```

Comment:
"Billing is based on the resources consumed, multiplied by the cost of that resource."

This just feels like an awkward phrasing. Might there be a better way to state this?